dicoogle/src/main/resources/webapp/js/components/search/result/imageView.js

```diff
@@ -142,7 +142,8 @@ const ImageView = createReactClass({
               // deprecated data properties
               "data-result-type": "image",
               "data-result-uri": item.uri,
-              "data-result-uid": item.sopInstanceUID
+              "data-result-uid": item.sopInstanceUID,
+              "data-result-serieuid": item.serieuid
```

Comment:
Despite the mistakes in the rest of the code, the singular form of _series_ is still _series_.

```suggestion
              "data-result-seriesuid": item.serieuid
```

---

This should not affect any of our WIP plugins.